hw_3/task_2.py

```diff
@@ -0,0 +1,11 @@
+#  Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+#  В результирующем списке не должно быть дубликатов.
+
+lst = [1, 2, 3, 4, 5, 1, 3, 7, 6, 4, 7]
+dublicate_lst = []
+
+for num in lst:
+    if lst.count(num) > 1:
+        if num not in dublicate_lst:
+            dublicate_lst.append(num)
```

Comment:
Чтобы понизить сложность и не использовать условие в 9 строке можно сделать так.
```suggestion
for num in set(lst):
    if lst.count(num) > 1:
        dublicate_lst.append(num)
```